src/main/java/com/blackducksoftware/integration/fortify/batch/util/MappingParser.java

```diff
@@ -86,7 +91,44 @@
             fe.printStackTrace();
```

Comment:
These exceptions need better handling. We can't just print stack traces. We at least need logging here.